Taskfile.yml

```
@@ -74,3 +74,25 @@ tasks:
       - cp ./docs/en/releases/template.ipynb ./docs/en/releases/jijmodeling-{{.CLI_ARGS}}.ipynb
       - 'yq -i ''.parts[-1].chapters |= [{"file": "releases/jijmodeling-{{.CLI_ARGS}}"}] + .'' docs/ja/_toc.yml'
       - 'yq -i ''.parts[-1].chapters |= [{"file": "releases/jijmodeling-{{.CLI_ARGS}}"}] + .'' docs/en/_toc.yml'
+
+  sync_paired_notebooks:
+    desc: Synchronize paired notebooks and markdown files
+    cmds:
+      - find docs \( -name "*.ipynb" -o -name "*.md" \) ! -name "template.ipynb" -exec uv run jupytext --sync {} \;
+
+  create_paired_notebooks:
+    desc: Create paired notebooks and markdown files
+    cmds:
+      - find docs \( -name "*.ipynb" -o -name "*.md" \) ! -name "template.ipynb" -exec uv run jupytext --set-formats ipynb,md:myst {} \;
+
+  check_paired_notebooks:
+    desc: Check the consistency of paired notebooks and markdown files
+    cmds:
+      - |
+        LOG=$(find docs \( -name "*.ipynb" -o -name "*.md" \) ! -name "template.ipynb" \
+          -exec uv run jupytext --pre-commit-mode --show-changes --sync {} \; \
+          2>&1 1>/dev/null)
+        if [ -n "$LOG" ]; then
```

こちらも `*.md` のみのマッチで十分だと思います。
また、少なくとも手元の環境では jupytext は log/diff/error 関係なく stdout に吐いているようで、これだと常に LOG が空になってしまってちゃんと差分を検出できませんでした。

そこで、JijModeling 本体側でやっているように、以下のように特定のエラーをマッチする事を提案します（ロバストかというと微妙ですが……）。

他の解決策としては、テモートで動かさず CI だけで実行するだけであれば、これの後に git diff を呼んで非自明な変更があれば落とすようにする、などという書き方ができて、こちらのほうが堅牢ではある（テモートだと commit していない変更がある場合動きませんが）かなと思います。

```suggestion
        LOG=$(find docs -name "*.md" \
          -exec uv run jupytext --pre-commit-mode --show-changes --sync {} \; )
        if [ -n "$(grep "Error: the git index is outdated." <<< "$LOG")" ]; then
```